Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -66,14 +66,26 @@ def from_string(self, string):
         return None
 
     def credentials(self, required=True):
-        return credentials(
+        args = dict(
             url=self.api_url,
             required=required,
             name=self.url.split('/')[2].replace('.', '_').upper(),
-            prompt="GitHub's API\nPlease generate a 'Personal access token' via 'Developer settings' for your user",
+            prompt='''GitHub's API
+Please generate a 'Personal access token' via 'Developer settings' with 'repo' and 'workflow'
+access for your {host} user
+https://{host}/settings/tokens'''.format(host=self.url.split('/')[2]),
             key_name='token',
         )
 
+        username, token = credentials(**args)
```

Comment:
This isn't a recursive call and calling webkitcorepy.credentials, right?
If so, might be a good idea to rename one of them, or explicitly call the webkitcorepy method.

Reply:
I'll make this an explicit `webkitcorepy.credentials` call.